samples/net/download/prj.conf

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+
+CONFIG_DOWNLOAD_CLIENT=y
+CONFIG_DOWNLOAD_CLIENT_STACK_SIZE=4096
+
+CONFIG_NEWLIB_LIBC=y
+
+# Networking
+CONFIG_NETWORKING=y
+# CONFIG_NET_SOCKETS_OFFLOAD=y
```

Comment:
I suggest setting this option in:
`nrf9160dk_nrf9160_ns.conf`

and removing `CONFIG_NET_SOCKETS_OFFLOAD=n` from the nrf7002dk_nrf53840_cpuapp.conf

Reply:
I agree with this, please remove the commented entry here, and put it in the project configuration files for the nRF9 series boards.